rosette/src/BigBang.cc

```diff
@@ -395,7 +395,7 @@ void InitBuiltinObs() {
 }
 
 
-static int is_absolute_pathname(char* path) { return path && path[0] == '/'; }
+//static int is_absolute_pathname(char* path) { return path && path[0] == '/'; }
```

Comment:
Please remove code instead of merely commenting it out. (Here and below)

Reply:
Will do.

Reply:
**Done**